frontend/src/PagesAdmin/RecruitmentSeparatePositionFormAdminPage/RecruitmentSeparatePositionFormAdminPage.tsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { SamfundetLogoSpinner } from '~/Components';
+import { SamfForm } from '~/Forms/SamfForm';
+import { SamfFormField } from '~/Forms/SamfFormField';
+import { getRecruitmentSeparatePosition, postRecruitmentSeparatePosition, putRecruitmentSeparatePosition } from '~/api';
+import { RecruitmentSeparatePositionDto } from '~/dto';
+import { STATUS } from '~/http_status_codes';
+import { KEY } from '~/i18n/constants';
+import { reverse } from '~/named-urls';
+import { ROUTES } from '~/routes';
+import styles from './RecruitmentSeparatePositionFormAdminPage.module.scss';
+import { AdminPageLayout } from '../AdminPageLayout/AdminPageLayout';
+
+type FormType = {
+  name_nb: string;
+  name_en: string;
+
+  description_nb: string;
+  description_en: string;
+
+  url: string;
+};
+
+export function RecruitmentSeparatePositionFormAdminPage() {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  // Form data
+  const { recruitmentId, separatePositionId } = useParams();
+  const [showSpinner, setShowSpinner] = useState<boolean>(true);
+  const [separatePosition, setSeparatePosition] = useState<Partial<RecruitmentSeparatePositionDto>>();
+
+  // Fetch data if edit mode.
+  useEffect(() => {
+    if (separatePositionId) {
+      getRecruitmentSeparatePosition(separatePositionId)
+        .then((data) => {
```

Comment:
```suggestion
        .then((response) => {
```
og endre resten av kallet til å bruke det navnet

Reply:
done